Tests/L0/ServiceFabricDeploy/AadDeploy.ps1

```diff
@@ -21,6 +21,7 @@ Register-Mock Get-VstsInput { $publishProfilePath } -- -Name publishProfilePath
 Register-Mock Get-VstsInput { $applicationPackagePath } -- -Name applicationPackagePath -Require
 Register-Mock Get-VstsInput { $serviceConnectionName } -- -Name serviceConnectionName -Require
 Register-Mock Get-VstsInput { "false" } -- -Name compressPackage
+Register-Mock Get-VstsInput { "SameAppTypeAndVersion" } -- -Name overwriteBehavior
```

Comment:
>"SameAppTypeAndVersion" [](start = 30, length = 23)

Should this be in a variable and reused in $publishArgs like $applicationPackagePath is?